mkdocs.yml

```diff
@@ -0,0 +1,227 @@
+      - Deleting content: guides/file/deleting_content.md
+      - Deleting a file: guides/file/deleting_file.md
+      - Sharing a file: guides/file/sharing_file.md
+      - Storage settings: guides/file/storage_settings.md
+    - Table:
+      - Creating a table: guides/table/create_table.md
+      - Managing columns: guides/table/managing_columns.md
+      - Storing an annotation: guides/table/storing_annotations.md
+      - Storing an activity: guides/table/storing_activities.md
+      - Getting metadata: guides/table/getting_metadata.md
+      - Downloading content: guides/table/downloading_content.md
+      - Uploding content: guides/table/uploading_content.md
+      - Deleting content: guides/table/deleting_content.md
+      - Deleting a table: guides/table/deleting_table.md
+      - Sharing a table: guides/table/sharing_table.md
+      - Storage settings: guides/table/storage_settings.md
+      - Creating a view: guides/table/creating_view.md
+  - API Reference:
+    # TODO:
+  # This API Reference section is meant to be a direct representation of the doc strings.
+  # That means we need to make sure that the doc strings are up to date and accurate.
+    - Models:
+    # todo: Restructure this section
+      - current:
+        - synapseclient: reference/synapseclient.md
+      - oop:
+        - Combined(Which looks best??): reference/oop/models.md
+        - Project: reference/oop/project.md
+        - Folder: reference/oop/folder.md
+        - File: reference/oop/file.md
+        - Table: reference/oop/table.md
+    - Command Line Interface: reference/cli.md
+  - Further Reading:
+    # TODO:
+    # Snippet of what synapse is, and links to further reading.
+    - What is Synapse?: nowhere.md
+    # What are use cases for annotations, how is it used in the real-world?
+    - Annotations: nowhere.md
+    # How does activity link to the concept of Provenance?
+    - Activity/Provenance: nowhere.md
+    # What storage options do we offer? S3, IBM, GCP ect...
+    - Storage Options: nowhere.md
+    # How is access control handled, how does it interact with things like open data?
+    - Access Control: nowhere.md
+    # Dive into how we handle versioning and anything that is 'cool to know'
+    - Versioning: nowhere.md
+    # What is an entity in terms of Synapse?
+    - Entity: nowhere.md
+    # Discuss what we are offering in the utils package
+    - SynapseUtils: nowhere.md
+    # Dive into how we handle caching for LRU cache and local file caching
+    - Caching: nowhere.md
+    # Dive into the manifest TSV for bulk actions
+    - Manifest Files: nowhere.md
+    # Dive deeper into how we handle the authentication on behalf of the user
+    - Authentication: nowhere.md
+    # Dive into the configuration file and how it works
+    - Configuration: nowhere.md
+  - News:
+    - news.md
+
+# Theme configuration
+theme:
+  name: material
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: teal
```

Comment:
It's hard to see the Synapse logo using this color:

![Screenshot 2023-12-11 at 4 20 45 PM](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/9377970/639e9a05-d8db-4753-9fa1-4f84beefe74a)

I tested some [other colors](https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#primary-color), and these work a little better:
* `white`
![Screenshot 2023-12-11 at 4 24 50 PM](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/9377970/fce195d1-ee7c-4dd6-8c07-80fb5edb97bc)
* `black`
![Screenshot 2023-12-11 at 4 25 11 PM](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/9377970/78996e8b-8aa8-4621-8f04-154df9e858b6)
* `cyan` (though the green part of the Sage logo gets a little lost)
![Screenshot 2023-12-11 at 4 25 26 PM](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/9377970/d62be5b3-6d63-4289-a743-2203d3e8fcd7)
* `yellow` - but I consider this chaotic good 🙃 
![Screenshot 2023-12-11 at 4 25 41 PM](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/9377970/84af031a-ef3c-4308-9741-e22c815c71fc)

We can also move forward with a custom color too.

Reply:
I think white for light theme and black for dark theme look pretty decent:
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/47131f32-7401-491d-8099-95b7ff2ae892)
![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/9792d850-a530-4400-b16f-b6ad58f7cb14)